File: code-coaches/medium/safety-deposit-boxes/SafetyDepositBoxes.cs
using System;

public class SafetyDepositBoxes
{
    static int CalculateTimeToGetItem(string[] items, string itemToFind)
    {
        int time = 5;
        foreach (string item in items)
        {
            if (item != itemToFind)
                time += 5;
            else
                break;
        }

        return time;
    }

    static void Main(string[] args)
    {
        string[] items = Console.ReadLine().Split(',');
        string itemToFind = Console.ReadLine();

        Console.WriteLine(CalculateTimeToGetItem(items, itemToFind));
    }
}

File: code-coaches/medium/safety-deposit-boxes/SafetyDepositBoxes.swift
func calculateTimeToGetItem(items: [String], itemToFind: String) -> Int {
    var time = 5
    for item in items {
        if item != itemToFind {
            time += 5
        } else {
            break
        }
    }

    return time
}

func main() {
    let itemsString = readLine()! 
	let itemToFind = readLine()!
	
	let items = itemsString.split(separator: ",").map { String($0) }
	print(calculateTimeToGetItem(items: items, itemToFind: itemToFind))
}

main()
